docs/tutorials/gpu-executor.md

```diff
@@ -26,6 +18,102 @@ This tutorial assumes you are already familiar with basic Jina concepts, such as
 If you're not yet familiar with these concepts, first read the [Basic Concepts](../fundamentals/concepts) and related documentation, and return to this tutorial once you feel comfortable performing baisc operations in Jina.
 ```
 
+## Jina & GPUs in a nutshell
+
+If you want a thorough walk-through of how to use GPU resources in your code, the full tutorial in the
+[next section](#Prerequisites) is exactly what you are looking for.
+
+But if you already know how to use your GPU and have come here just to find out how to make it play nice with Jina,
+then we have good news for you:
+You just use your GPU like you usually would in your machine learning framework of choice, and you are off to the races.
+Jina enables you to use GPUs like you normally would in a Python script, or in a Docker 
+container - it does not impose any additional requirements or configuration.
+
+
+Let's take a look at a minimal working example, written in PyTorch.
+
+```python
+
+import torch
+from jina import DocumentArray, requests
+
+class MyGPUExec(Executor):
+    def __init__(self, device: str = 'cpu', *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.device = device
+
+
+    @requests
+    def encode(self, docs: Optional[DocumentArray], **kwargs):
+        with torch.inference_mode():
+          ## generate random embeddings
+          embeddings = torch.rand((len(docs),5), device=self.device)
```

Comment:
Can we have some example that actually use a `model` that uses device? Here I just have to believe they are created but I cannot feel

Reply:
That is done further down in the tutorial, using a sentence encoder, i kept that more or less as it was in the previous version. This first part is more intended for people that already know how to use their model on GPU, and just want to quickly see how to integrate that with jina. The second part is more elaborate.